javers-core/src/test/groovy/org/javers/core/JaversRepositoryE2ETest.groovy

```diff
@@ -649,4 +649,36 @@ class JaversRepositoryE2ETest extends Specification {
       snapshots[1].version == 1
 
     }
+
+    def "should query for Entity snapshot with given commit id in a form of CommitId"() {
```

Comment:
no właśnie nie wiem czy tak ładnie się da. Próbowałem to zrobić wcześniej ale uznałem że dwa oddzielne testy są bardziej czytelne. Dla porównania zaraz dorzucę commita ze zwiniętą wersją.